configureWorkspace/configure.ts

```diff
@@ -153,15 +153,17 @@ LABEL Name=${serviceName} Version=${version}
 EXPOSE ${port}
 
 WORKDIR /app
-ADD . /app
+COPY . .
+
+# install dependencies if dependencies file is present
```

Comment:
This comment I think is implied by the echo statement, can you move the RUN command back under the " Using pip" comment?